deepfence_server/pkg/integration/integration.go

```diff
@@ -58,17 +58,32 @@ func GetIntegration(ctx context.Context, integrationType string, b []byte) (Inte
 func IsMessagingFormat(integrationType string) bool {
 	retVal := false
 	switch integrationType {
-	case constants.Slack, constants.Teams, constants.PagerDuty,
-		constants.Email, constants.Jira:
+	case constants.Slack,
+		constants.Teams,
+		constants.PagerDuty,
+		constants.Email,
+		constants.Jira:
 		retVal = true
 	}
 	return retVal
 }
 
+func SupportsSummaryLink(integrationType string) bool {
+	switch integrationType {
+	case constants.Slack,
+		constants.Teams,
+		constants.Email:
+		return true
+	}
+	return false
```

Comment:
nit: I think we should maintain a array of supported in constants. feel free to ignore this comment

Reply:
will keep this for now